app/services/user-skills-list.js

```diff
@@ -0,0 +1,55 @@
+import Ember from 'ember';
+import skillsList from 'code-corps-ember/utils/skills-list';
+
+const {
+  computed: {
+    alias, empty
+  },
+  get,
+  getProperties,
+  inject: { service },
+  isEmpty,
+  Service
+} = Ember;
+
+export default Service.extend({
+  currentUser: service(),
+  store: service(),
+
+  isEmpty: empty('userSkills'),
+  user: alias('currentUser.user'),
+  userSkills: alias('user.userSkills'),
+
+  add(skill) {
+    let { store, user } = getProperties(this, 'store', 'user');
+    return store.createRecord('user-skill', { user, skill }).save();
+  },
+
+  contains(skill) {
+    let userSkills = get(this, 'userSkills');
+    return skillsList.contains(userSkills, skill);
```

Comment:
ah yes - I definitely read this as if it was `Enumerable#contains` at first. But that makes sense now.